app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -71,9 +111,44 @@ export default Controller.extend(ModalFunctionality, {
     }
   },
 
+  bindKeyboardShortcuts() {
+    if (!this.mouseTrap) {
+      this.mouseTrap = Mousetrap(
```

Comment:
I don't like using global modules like this. Instead I would prefer if you exposed an API in the `keyboard-shortcuts` module that would allow you to do what you need.

For example:

```
KeyboardShortcuts.pause(GLOBAL_SHORTCUTS_TO_PAUSE);
KeyboardShortcuts.addBindings(BOOKMARK_BINDINGS);

// and then to restore:

KeyboardShortcuts.removeBindings(BOOKMARK_BINDINGS);
KeyboardShortcuts.unpause(GLOBAL_SHORTCUTS_TO_PAUSE);
```